main.py

```
@@ -1,609 +1,802 @@
+        if not isinstance(k, str) or not isinstance(v, str):
+            continue
+        vv = v.strip()
+        if not vv:
+            continue
+        set_credential(k.strip(), vv)
+        updated.append(k.strip())
+    return {"updated": updated}
+
+
+@app.delete("/settings/api-keys")
+def clear_api_keys(request: ApiKeysClearRequest = Body(default=ApiKeysClearRequest())):
```

Ruff (0.15.10)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)